src/test/java/teammates/test/cases/testdriver/TestngTest.java

```diff
@@ -0,0 +1,75 @@
+package teammates.test.cases.testdriver;
+
+import static org.testng.AssertJUnit.assertTrue;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.testng.annotations.Test;
+
+import teammates.common.util.FileHelper;
+import teammates.test.cases.BaseTestCase;
+
+public class TestngTest extends BaseTestCase {
+
+    @Test
+    public void checkTestsInTestng() throws FileNotFoundException {
+        HashMap<String, String> tests = new HashMap<String, String>(); // <class name, package name>     
+        String testngXmlAsString = FileHelper.readFile("./src/test/testng.xml");
+        
+        addFilesToTestsRecursively(tests, "./src/test/java/teammates/test/cases", true, "", 
```

Comment:
Try to make this method return something, e.g `HashMap<String, String> tests = addFilesToTestsRecursively(new HashMap<String, String>(), ...)`. It's more testable that way.